tcms/testplans/static/testplans/js/search.js

```diff
@@ -189,9 +209,44 @@ export function pageTestplansSearchReadyHandler () {
         return false // so we don't actually send the form
     })
 
+    $('#btn_clone_bulk').click(function () {
+        const selectedTestPlans = getSelectedTestPlans()
+        if (selectedTestPlans.length === 0) {
+            alert('No test plans selected for cloning.')
+            return false
+        }
+
+        window.location.assign(`/plan/clone/?p=${selectedTestPlans.join('&p=')}`)
+    })
+
     $('#id_product').change(updateVersionSelectFromProduct)
 }
 
+function getSelectedTestPlans () {
```

Comment:
hmmm, this feels like a duplicate. Need to check if we don't have something more generic which already does this. 

In testplans/static/testplans/js/get.js there is getSelectedTestCases() and IIRC there was another one in utils.js but let's leave this for later.